app/views/templates/package_action.erb

```diff
@@ -0,0 +1,46 @@
+<%#
+kind: job_template
+name: Package Action - SSH Default
+job_name: PackageAction
```

Comment:
Not really, that does looks better.  In my head I was thinking of it more like an identifier.